tests/dummy/config/deprecation-workflow.js

```diff
@@ -2,5 +2,6 @@ window.deprecationWorkflow = window.deprecationWorkflow || {};
 window.deprecationWorkflow.config = {"workflow":[
   { matchMessage: 'silence-me', handler: 'silence' },
   { matchMessage: 'log-me', handler: 'log' },
-  { matchMessage: 'throw-me', handler: 'throw' }
+  { matchMessage: 'throw-me', handler: 'throw' },
+  { idMatch: 'ember.workflow', handler: 'log' }
```

Comment:
Let's tweak this to `matchId` (to keep with the same pattern as `matchMessage`).